string place
string transition
string token
string fire_guard
---
string place
string transition
string token
string fire_guard
